test-suite/tests/compact-p002-out.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+  "@context": {
+    "ex": {"@id": "http://example.org/"}
+  },
+  "@id": "http://example.org/id1",
+  "@type": ["http://example.org/Type1", "http://example.org/Type2"],
+  "http://example.org/term": {"@id": "http://example.org/id2"}
+}
```

Comment:
Shouldn't `ex` be used as a prefix definition here and thus be used in the compaction? Its value ends with a "/". I would expect a (negative) test for this to leave terms as-is (as IRIs) when a simple term does *not* end with a gen-delim or the key with a ":". Or am I missing something?